### PR TITLE
Split app info card and fix stylings

### PR DESCRIPTION
**Pull request by SeverS**
## Description

Mostly component and styling refactorings required for the extension page pr.

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

**Review by josenriagu — APPROVED** [review]

File: libs/design-system-components/src/components/ExtensionImageGallery/index.tsx [hunk 1]
```diff
@@ -34,21 +33,27 @@ const ExtensionImageGallery: React.FC<ExtensionImageGalleryProps> = props => {
...
-        {images.map((image, index) => (
+      <div className={images?.length === 1 ? tw('flex') : tw(`${gridStyle}`)}>
+        {images?.map((image, index) => (
           <ImageBlockGridItem
-            image={image}
+            image={{ src: image.src, size: { width: image.width, height: image.height } }}
             key={index}
             handleClickImage={handleClickImage}
-            images={images}
+            images={images.map(img => ({
```
Comment (kenshyx) [inline libs/design-system-components/src/components/ExtensionImageGallery/index.tsx:42]: shouldn't here be also optional chaining for `?.map`?

File: extensions/apps/extensions/src/components/pages/info-page/info-page.tsx [hunk 2]
```diff
@@ -155,7 +162,7 @@ export const InfoPage: React.FC<InfoPageProps> = ({ appId }) => {
                   displayName={appData.displayName}
                   extensionType={appData.applicationType}
                   // @todo: do we have a placeholder app logo?
-                  extensionIconSrc={transformSource(appData.logoImage?.src)?.src}
+                  extensionAvatar={appData.logoImage}
```
Comment (kenshyx) [inline extensions/apps/extensions/src/components/pages/info-page/info-page.tsx:165]: logoImage is optional

File: libs/design-system-components/src/components/ExtensionImageGallery/index.tsx (hunk 1 above)
Comment (SeverS) [reply libs/design-system-components/src/components/ExtensionImageGallery/index.tsx:42]: there is an optional chaining at line 37

File: extensions/apps/extensions/src/components/pages/info-page/info-page.tsx (hunk 2 above)
Comment (SeverS) [reply extensions/apps/extensions/src/components/pages/info-page/info-page.tsx:165]: it is handled by the AppInfoHeader

File: extensions/apps/extensions/src/components/pages/info-page/info-page.tsx [hunk 3]
```diff
@@ -119,6 +119,13 @@ export const InfoPage: React.FC<InfoPageProps> = ({ appId }) => {
     }
   }, [appData?.applicationType, t]);
 
+  const coverImageSrc = useMemo(() => {
```
Comment (kenshyx) [inline extensions/apps/extensions/src/components/pages/info-page/info-page.tsx:122]: can you please add some comments here explaining the usecase? Why it returns `/public?`

Comment (SeverS) [reply extensions/apps/extensions/src/components/pages/info-page/info-page.tsx:122]: that is something that i still need to fix, I need to add the resources to the ds-core first

File: libs/design-system-components/src/components/ExtensionImageGallery/index.tsx (hunk 1 above)
Comment (kenshyx) [reply libs/design-system-components/src/components/ExtensionImageGallery/index.tsx:42]: I didn't see the shadowing, so every ImageBlockGridItems receives the entire images list?

File: extensions/apps/extensions/src/components/pages/info-page/info-page.tsx [hunk 4]
```diff
@@ -63,151 +82,285 @@ export const InfoPage: React.FC<InfoPageProps> = ({ appId }) => {
...
+                      <Text variant="body2" color={{ light: 'grey4', dark: 'grey7' }}>
+                        {t('Latest update')}
+                      </Text>
+                      <Button
+                        variant="text"
+                        size="md"
+                        label={formatRelativeTime(latestRelease?.node?.createdAt)}
+                        onClick={() => {
+                          navigate({
+                            to: '/info/$appId/audit-log',
```
Comment (kenshyx) [inline extensions/apps/extensions/src/components/pages/info-page/info-page.tsx:275]: is this path handled? what should it display?

Comment (SeverS) [reply extensions/apps/extensions/src/components/pages/info-page/info-page.tsx:275]: all the navigation to the subroutes is broken... you can check this better on next. I need to debug to see what is going on

Comment (SeverS) [reply extensions/apps/extensions/src/components/pages/info-page/info-page.tsx:275]: also the subroutes will not be included in the current task, it's just too much to add there but I'll try to at least fix the routing

**Pull request merged**